app/views/content/improve-your-subject-knowledge.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Improve your subject knowledge
+image: "/assets/images/international-dt.jpg"
+article_classes: ['longform']
+mobileimage: "/assets/images/steps-hero-mob.jpg"
+description: |-
+  School experience can help you talk to teachers and get an insight into day-to-day school life. Here's how to get school experience.
+date: "2021-05-04"
+right_column:
+  ctas:
+    - title: Find and apply for teacher training
+      link_text: "Find postgraduate teacher training courses"
+      link_target: "https://www.gov.uk/find-postgraduate-teacher-training-courses"
+      icon: "icon-arrow"
+      hide_on_mobile: Yes
+      hide_on_tablet: Yes
+keywords:
+  - Subject knowledge
+  - Subject knowledge enhancement
+  - SKE
+---
+If you're concerned about how well you know the subject you want to teach, do not worry - you may be able to do a 'subject knowledge enhancement' course.
```

Comment:
I wonder if having the abbreviation somewhere on the page might help, if only for SEO etc.